packages/notebook/src/widget.ts

```diff
@@ -1250,6 +1250,9 @@ export namespace StaticNotebook {
  * A notebook widget that supports interactivity.
  */
 export class Notebook extends StaticNotebook {
+
+  footer: NotebookFooter;
```

Comment:
```suggestion
  protected footer: NotebookFooter;
```

Reply:
Ah, for this to be accessible from `NotebookActions`, it actually has to be public I think.

Reply:
Well, it does not need to be there at all since we can use layout directly, see https://github.com/jupyterlab/jupyterlab/pull/14796/commits/209612599eeb32d1759a486aaf3ffbf1b23068ee.